05-geometry-operations.Rmd

```diff
@@ -502,12 +501,17 @@ names(sfs_st_cast) = abbreviate_geomtypes(names(sfs_st_cast))
 names(sfs_st_cast)[1] = ""
 knitr::kable(sfs_st_cast, 
```

Comment:
@Nowosad pls check if you are happy with the updated table (I think you have created it originally). I am not sure if I understand the columns. Have you chosen randomly the numbers for the `POI` column or where do these numbers come from? Maybe I am missing something, if not, we should clarify the origin of these numbers.